src/main/scala/edu/cmu/cs/obsidian/codegen/FuncScope.scala

```diff
@@ -5,17 +5,13 @@ class FuncScope(f: FunctionDefinition) {
     class Body(val code: String){}
```

Comment:
This class could use some more comments. Why "FuncScope" — it seems to describe a whole function? And why is arg 0 special?

Reply:
This is old code that I didn't write, so I also don't know why it's named that and wish it had more comments. Over time I've been winnowing it down and deleting dead code or making it more direct until it's gotten quite small. The name matches with "ObjScope" in the other file, but I don't know why either was picked. It's part of how we output things to a file via mustache; `arg0` is special because of how it's treated in `functions.mustache`.